assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
+}
+p {
+  font-size: 1.25em;
+  color: gray;
+  line-height: 32px;
+}
+
+h1 {
+  font-size: 40px !important;
+  color: black;
+}
+
+.white-text p{
+  color: white;
+}
+.white-text h1{
+  color: white;
+}
+
+.divider-text {
+  padding: 40px 50px 50px 50px;
+}
+
+/* Menu Logo */
+
+h1.top-logo{
+  background: url("../img/logo.png") no-repeat 0 0;
+  margin-left: 18px;
+}
+h1.top-logo a{
+  display:block;
+  width: 200px;
+  text-indent:-9999px;
+}
+
+@media(max-width: 768px) {
```

Comment:
Faaaaantastic work! I wasn't expecting you to make this site responsive out of the gate.